action/protocol/rewarding/fund.go

```diff
@@ -165,10 +171,6 @@ func (p *Protocol) AvailableBalance(
 
 // DepositGas deposits gas into the rewarding fund
 func DepositGas(ctx context.Context, sm protocol.StateManager, amount *big.Int, opts ...protocol.Option) ([]*action.TransactionLog, error) {
-	// If the gas fee is 0, return immediately
-	if amount.Cmp(big.NewInt(0)) == 0 {
-		return nil, nil
-	}
```

Comment:
why remove this now?

Reply:
make sure options is also executed when amount is 0

Reply:
as we discussed, now we have `baseFee` which is always non-zero and add into rewarding pool, so I think we can revert this? or revert in new PR4394